### DISCUSSION TITLE
How do I add custom indentation rules for specific symbols using the structural editor?

### DISCUSSION THREAD

**Discussion by TuggyNE** [BetterThanTomorrow/calva#1805] [Q&A]
The description of `calva.fmt.newIndentEngine` implies that cljfmt is likely to be unsupported in Calva at some point. I'd like to get started on some custom indentation rules for various 3rd-party symbols (including some I've customized clj-kondo linting for, especially `if-let`-alikes), but I can't find any documentation for how to get started. Should I just use cljfmt for now?

**Comment by bpringe** [comment]
I think cljfmt is the way to go for this. @PEZ can you confirm?
